client/components/Dashboard.react.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <div className="ui grid container">
+        <div className="row">
+          <div className="column">
+            <div className="ui six buttons">
+              <button className="ui button">All</button>
+              <button className="ui button">Period 1</button>
+              <button className="ui button">Period 2</button>
+              <button className="ui button">Period 3</button>
+              <button className="ui button active">Period 7</button>
+              <button className="ui button">Period 8</button>
+            </div>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="column">
+            <h1 className="ui header">Period 7</h1>
+            <div className="column">
+              <table className="ui selectable celled table large table">
+                <thead>
+                  <tr>
+                    <th className="five wide">Student</th>
+                    <th className="three wide">
+                      Total Time Spent Outside Your Class This Year
+                    </th>
+                    <th className="eight wide">Today</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td>John</td>
+                    <td>3 min</td>
+                    <td />
+                  </tr>
+                  <tr>
+                    <td>Jamie</td>
```

Comment:
Let's react-ify the students - create a separate component called `StudentEntryRow.react.js` and have it take in props for `name`, `timeTotal`, and `timeToday` and return the same `<tr>` block, except using the props.  Also include `cssClass` or something as a prop so we can also include `className` assignments like you did below on the `<tr>` block.

Reply:
Should we reactify as much as we can? It looks like the UI buttons could a component too.